interactive_continue_printer.go

```diff
@@ -0,0 +1,183 @@
+		DefaultValueIndex: 0,
+		DefaultText:       "Do you want to continue",
+		TextStyle:         &ThemeDefault.PrimaryStyle,
+		Options:           []string{"yes", "no", "all", "stop"},
+		OptionsStyle:      &ThemeDefault.SuccessMessageStyle,
+		SuffixStyle:       &ThemeDefault.SecondaryStyle,
+	}
+)
+
+// InteractiveContinuePrinter is a printer for interactive continue prompts.
+type InteractiveContinuePrinter struct {
+	DefaultValueIndex int
+	DefaultText       string
+	TextStyle         *Style
+	Options           []string
+	OptionsStyle      *Style
+	Handles           []string
+	ShowshortHandles  bool
+	SuffixStyle       *Style
+}
+
+// WithDefaultText sets the default text.
+func (p InteractiveContinuePrinter) WithDefaultText(text string) *InteractiveContinuePrinter {
+	p.DefaultText = text
+	return &p
+}
+
+// WithDefaultValueIndex sets the default value, which will be returned when the user presses enter without typing any letter.
+func (p InteractiveContinuePrinter) WithDefaultValueIndex(value int) *InteractiveContinuePrinter {
+	if value >= len(p.Options) {
+		panic("Index out of range")
+	}
+	p.DefaultValueIndex = value
+	return &p
+}
+
+// WithDefaultValue sets the default value, which will be returned when the user presses enter without typing any letter.
+func (p InteractiveContinuePrinter) WithDefaultValue(value string) *InteractiveContinuePrinter {
+	for i, o := range p.Options {
+		if o == value {
+			p.DefaultValueIndex = i
+			break
+		}
+	}
+	return &p
+}
+
+// WithTextStyle sets the text style.
+func (p InteractiveContinuePrinter) WithTextStyle(style *Style) *InteractiveContinuePrinter {
+	p.TextStyle = style
+	return &p
+}
+
+// WithOptions sets the options.
+func (p InteractiveContinuePrinter) WithOptions(options []string) *InteractiveContinuePrinter {
+	p.Options = options
+	return &p
+}
+
+// WithHandles allows you to customize the short handles for the answers.
+func (p InteractiveContinuePrinter) WithHandles(handles []string) *InteractiveContinuePrinter {
+	if len(handles) != len(p.Options) {
+		panic("Invalid number of handles")
+	}
+	p.Handles = handles
+	return &p
+}
+
+// WithShortHandles will set ShowShortHandles to true
+// this makes the printer display the shorthand options instead their shorthand version.
+func (p InteractiveContinuePrinter) WithShortHandles() *InteractiveContinuePrinter {
+	p.ShowshortHandles = true
+	return &p
+}
+
+// WithOptionsStyle sets the continue style.
+func (p InteractiveContinuePrinter) WithOptionsStyle(style *Style) *InteractiveContinuePrinter {
+	p.OptionsStyle = style
+	return &p
+}
+
+// WithSuffixStyle sets the suffix style.
+func (p InteractiveContinuePrinter) WithSuffixStyle(style *Style) *InteractiveContinuePrinter {
+	p.SuffixStyle = style
+	return &p
+}
+
+// Show shows the continue prompt.
+//
+// Example:
+//
+//	 result, _ := pterm.DefaultInteractiveContinue.Show("Do you want to apply the changes?")
+//		pterm.Println(result)
```

Comment:
```suggestion
//	result, _ := pterm.DefaultInteractiveContinue.Show("Do you want to apply the changes?")
//	pterm.Println(result)
```

Reply:
This formatting is actually coming from `gopls`, I use the more strict mode and when I save the file it formats these...